tests/migrations/test_commands.py

```diff
@@ -337,6 +337,124 @@ def test_showmigrations_plan_squashed(self):
+        "migrations.migrations_test_apps.alter_fk.book_app",
+    ])
+    def test_showmigrations_plan_single_app_label(self):
+        """
+        Tests `--plan app_label` output of `showmigrations`.
+        """
+        # Single app with no dependencies on other apps.
+        out = six.StringIO()
+        call_command("showmigrations", "mutate_state_b", format='plan', stdout=out)
+        self.assertEqual(
+            "[ ]  mutate_state_b.0001_initial\n"
+            "[ ]  mutate_state_b.0002_add_field\n",
+            out.getvalue()
+        )
+
+        # Single app with dependencies.
+        out = six.StringIO()
+        call_command("showmigrations", "author_app", format='plan', stdout=out)
+        self.assertEqual(
+            "[ ]  author_app.0001_initial\n"
+            "[ ]  book_app.0001_initial\n"
+            "[ ]  author_app.0002_alter_id\n",
+            out.getvalue()
+        )
+
+        # Some migrations already applied.
+        call_command("migrate", "author_app", "0001", verbosity=0)
+        out = six.StringIO()
+        call_command("showmigrations", "author_app", format='plan', stdout=out)
+        self.assertEqual(
+            "[X]  author_app.0001_initial\n"
+            "[ ]  book_app.0001_initial\n"
+            "[ ]  author_app.0002_alter_id\n",
+            out.getvalue()
+        )
+
+        # Cleanup by unmigrating author_app.
+        call_command("migrate", "author_app", "zero", verbosity=0)
+
+    @override_settings(INSTALLED_APPS=[
+        "migrations.migrations_test_apps.mutate_state_b",
+        "migrations.migrations_test_apps.alter_fk.author_app",
+        "migrations.migrations_test_apps.alter_fk.book_app",
+    ])
+    def test_showmigrations_plan_multiple_app_labels(self):
+        """
+        Tests --plan output of `showmigrations`
+        with multiple app_labels provided.
+        """
+        # Multiple apps: author_app depends on book_app; mutate_state_b doesn't depend on other apps.
+        out = six.StringIO()
+        call_command("showmigrations", "mutate_state_b", "author_app", format='plan', stdout=out)
+        self.assertEqual(
+            "[ ]  author_app.0001_initial\n"
+            "[ ]  book_app.0001_initial\n"
+            "[ ]  author_app.0002_alter_id\n"
+            "[ ]  mutate_state_b.0001_initial\n"
+            "[ ]  mutate_state_b.0002_add_field\n",
+            out.getvalue()
+        )
+        # Multiple apps: args order shouldn't matter (the same result is expected as above).
+        out = six.StringIO()
+        call_command("showmigrations", "author_app", "mutate_state_b", format='plan', stdout=out)
+        self.assertEqual(
+            "[ ]  author_app.0001_initial\n"
+            "[ ]  book_app.0001_initial\n"
+            "[ ]  author_app.0002_alter_id\n"
+            "[ ]  mutate_state_b.0001_initial\n"
+            "[ ]  mutate_state_b.0002_add_field\n",
+            out.getvalue()
+        )
```

Comment:
Since the same result is expected as above, wouldn't it be clearer to save the output in a variable? Something like:
```
expected = (
            "[ ]  author_app.0001_initial\n"
            "[ ]  book_app.0001_initial\n"
            "[ ]  author_app.0002_alter_id\n"
            "[ ]  mutate_state_b.0001_initial\n"
            "[ ]  mutate_state_b.0002_add_field\n"
)